docs/03-node-operators/configurations/fiatStableMinGasPrice/fiat-stable-min-gas-price.md

```diff
@@ -0,0 +1,148 @@
+- **MinGasPrice**: This would be the **minimum** allowed gas price — the lowest cost you can set for your transaction to be accepted or processed.
+
+**So together:**
+
+**Fiat Stable MinGasPrice** means setting a minimum gas price that, no matter how much RBTC's price goes up or down, tries to represent a *fixed real-world value* (like "always costing about 0.01 USD worth of gas minimum").
+
+### Why does it matter?
+
+- If RBTC price suddenly spikes, gas fees could become ridiculously high in dollar terms.
+- By using a **fiat-stable minimum**, systems can ensure fair and predictable pricing for users and maintain network security and validator incentives.
+
+### Motivation and context for the changes
+
+Currently, RSK miners can set the minimum gas price (MinGasPrice) using the miner.minGasPrice configuration, which is denominated in WEI and inherently linked to the Bitcoin price. This linkage causes transaction costs to fluctuate with Bitcoin's value, potentially leading to increased costs when Bitcoin appreciates, thereby adversely affecting user experience.
+
+To address this issue, the proposed feature aims to empower miners with the ability to specify and configure the minimum gas price in fiat currency. This change would stabilize transaction costs in fiat terms, regardless of cryptocurrency market variations, and is intended to enhance predictability and user satisfaction. The system used to support only one configuration parameter for setting the MinGasPrice. See an example [here](https://www.notion.so/Minimum-Gas-Price-Feature-Validation-on-Testnet-122c132873f9807ba817f63a13aa7720?pvs=21),
+
+The goal is to give miners a capability to specify / configure min gas price value denominated in a fiat currency.
+
+### PR that added the changes
+
+[https://github.com/rsksmart/rskj/pull/2310](https://github.com/rsksmart/rskj/pull/2310)
+
+### **🔗 HTTP Call (Off-Chain Method)**
+
+This approach uses a **centralized or semi-centralized API** to fetch the current **RBTC-to-fiat price** and then calculate a gas price based on that.
+
+### How it works:
+
+- The system (like a wallet, dApp backend, or node) sends an **HTTP request** to a price oracle or API like:
+    - CoinGecko
+    - Chainlink's external adapters
+    - Custom backend service
+- It retrieves the current **RBTC/USD price**, then calculates:
+  `minGasPrice = fiatMinPrice / ethPrice`
+- Example: If you want the gas price to represent $0.01, and RBTC = $2000, then:
+  `minGasPrice = 0.01 / 2000 = 0.000005 RBTC (or 5 gwei)`
+
+### ✅ Pros:
+
+- Simple, flexible, easy to integrate.
+- Can pull from a wide range of data sources.
+
+### ❌ Cons:
+
+- **Centralization risk** – you're trusting the HTTP source.
+- **Not trustless** – the blockchain doesn't verify the result.
+- **Latency and reliability** – dependent on network and endpoint availability.
+
+---
```

Comment:
I suggest we get rid of this "---" marker and leave only one space between the previous text and the next title.

Reply:
Sounds good